### DISCUSSION TITLE
How to sort topics or choose a different order other than alphabetically?

### DISCUSSION THREAD

**Discussion by OscarCarrilloTR** [community/community#116589] [New to GitHub]
I just review this page:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics

But I can't find how to put the topics in a specific order other than alphabetically, in my current project it's this way but I've seen some projects that these are not sorted this order (alphabetically).

For instace look at this repository:
![imagen](https://github.com/community/community/assets/91084534/a5e46974-d973-4c44-997c-ba01b4115e03)

**Comment by jfullstackdev** [comment]
Copilot said it's maybe granted special permission to have such, as this is a famous curated lists but you can always contact some contributor in there to verify it

**Reply by devashish2024** [reply]
bruh spamming copilot :skull:

**Reply by jfullstackdev** [reply]
what is your problem ? look at my profile if you think I'm a spam and here is the total list of those I helped, so if you are the one that cannot offer help, you should be the one to shut up

https://github.com/orgs/community/discussions?discussions_q=commenter%3Ajdevstatic+

**Reply by OscarCarrilloTR** [reply]
I assume that maybe in the past there was ordered by the default order and later in an update GitHub started to sort them automatically in alphabetical order.

**Reply by devashish2024** [reply]
Ok I'm sorry but you should be polite.

_edit: if you are polite, dont reply me a single comment back_
